docs/github-sync.md

```diff
@@ -0,0 +1,56 @@
+# GitHub Sync
+
+This extension implemnents GitHub Sync, allowing you to synchronize specific pages directly from Roam to a specified GitHub repository as Issues.
+
+## Config Page
+
+This will be found at `[[roam/js/github-sync]]` in your graph.
```

Comment:
Initially this felt far enough away from Discourse Graph to be on it's own.  Then I ended up hooking into Discourse Nodes for their definition 😅.  But I could see users want to use only this feature and none of the other dgraph features ... then it would be kind of buried / confusing.